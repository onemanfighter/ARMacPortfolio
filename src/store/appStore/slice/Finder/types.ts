export type FinderState = {};

export interface FinderAppAction {}

export type FinderStateSlice = FinderState & FinderAppAction;
