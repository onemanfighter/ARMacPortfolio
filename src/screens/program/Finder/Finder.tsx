import { FinderProps } from './type';

const Finder = (props: FinderProps) => {
  return (
    <div>
      <h1>Finder</h1>
    </div>
  );
};

export default Finder;
