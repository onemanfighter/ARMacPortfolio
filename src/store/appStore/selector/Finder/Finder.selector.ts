import { AppStoreState } from '../../appStore';

const finderSelector = (state: AppStoreState) => ({});

export { finderSelector };
