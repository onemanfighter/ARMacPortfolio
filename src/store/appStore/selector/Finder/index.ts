export { finderSelector } from './Finder.selector';
