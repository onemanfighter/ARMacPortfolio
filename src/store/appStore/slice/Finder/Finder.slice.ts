import { AppStoreSlice } from '../../appStore';
import { FinderState, FinderStateSlice } from './types';

const defaultFinderState: FinderState = {};

const createFinderSlice: AppStoreSlice<FinderStateSlice> = (set) => ({
  ...defaultFinderState,
});

export default createFinderSlice;
