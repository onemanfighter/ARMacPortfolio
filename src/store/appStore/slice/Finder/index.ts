export { default as createFinderSlice } from './Finder.slice';
export type { FinderStateSlice, FinderState } from './types';
