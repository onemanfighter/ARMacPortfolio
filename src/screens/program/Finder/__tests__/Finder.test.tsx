import { render } from '@testing-library/react';
import Finder from '../Finder';

describe('Finder', () => {
  it('should render correctly to match snapshot', () => {
    const { container } = render(<Finder />);

    expect(container).toMatchSnapshot();
  });
});
