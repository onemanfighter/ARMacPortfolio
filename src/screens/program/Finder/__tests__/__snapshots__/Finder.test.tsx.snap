// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Finder should render correctly to match snapshot 1`] = `
<div>
  <div>
    <h1>
      Finder
    </h1>
  </div>
</div>
`;
